### PR TITLE
Show uncommitted message before displaying profile

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2515.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- create a new model
- click on the header logo
- you should go to the profile
- create a new model
- add an app
- click on the header logo
- you should see the warning about losing changes

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6850/

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -1260,16 +1260,13 @@ YUI.add('juju-gui', function(Y) {
         if (!auth) {
           return;
         }
-        this.state.changeState({
-          model: null,
-          profile: auth.rootUserName,
-          root: null,
-          store: null
-        });
+        views.utils.showProfile(
```
Comment (frankban) [inline jujugui/static/gui/src/app/app.js]: Ah, good that we reuse this existing method. Maybe we should set the store field to null as well in utils._showProfile?

Comment (huwshimi) [inline jujugui/static/gui/src/app/app.js]: Good call, I missed that one.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6851/

**Comment by hatched** [comment]
Thanks @huwshimi 
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**